src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1399,7 +1399,7 @@ private void CheckModelAndSyntaxNodeToSpeculate(CSharpSyntaxNode syntax)
         /// scope around position is used.</param>
         /// <param name="name">The name of the symbol to find. If null is specified then symbols
         /// with any names are returned.</param>
-        /// <param name="includeReducedExtensionMethods">Consider (reduced) extension methods.</param>
+        /// <param name="includeReducedExtensionMethods">Consider extension members. Classic extension methods will be returned in reduced form.</param>
```

Comment:
Should the parameter be also renamed?
```suggestion
        /// <param name="includeReducedExtensionMembers">Consider extension members. Classic extension methods will be returned in reduced form.</param>
```

Reply:
Sadly, this name is part of the public API, so changing it would be a break for any consumers that use an explicit argument name. That's why I left it. I can mark it as PROTOTYPE and we can revisit as part of API discussion

Reply:
Oops, sorry, you're talking about the `CSharpSemanticModel` one which is internal. Yes, I'll do that